Purchases/Purchasing/OfferingsFactory.swift

```diff
@@ -17,15 +17,14 @@ import StoreKit
 
 class OfferingsFactory {
 
-    func createOfferings(fromProductDetailsByID productDetailsByID: [String: ProductDetails],
-                         data: [String: Any]) -> Offerings? {
+    func createOfferings(from storeProductsByID: [String: StoreProduct], data: [String: Any]) -> Offerings? {
         guard let offeringsData = data["offerings"] as? [[String: Any]] else {
             return nil
         }
 
         let offerings = offeringsData.reduce([String: Offering]()) { (dict, offeringData) -> [String: Offering] in
             var dict = dict
-            if let offering = createOffering(fromProductDetailsByID: productDetailsByID,
+            if let offering = createOffering(from: storeProductsByID,
```

Comment:
I simplified by doing just `from:` instead of `fromStoreProductsByID:`, but happy to revert if the latter feels clearer to others